### PR TITLE
Release 1.0.8

### PR DESCRIPTION
**Pull request by thesamim**
Take care of a truckload of issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,112 +2,141 @@
...
 ## Notes
-1. TickTick lists are referred to as Projects in the plugin, and throughout documentation.
-2. TickTick only has four priorities (High, Medium, Low, None). Please see Priority in [Task Format](#Task-format) for Task mapping of priority
-3. Please see known issue 3. below. __Strongly__ recommend using subtasks rather than items to avoid loss of data.
 
+1. **All Task Deletion operations must be confirmed.** In the fullness of time, when we get out of Beta, this might be a preference item. For now it's compulsory.
+2. TickTick lists are referred to as Projects in the plugin, and throughout documentation.
+3. TickTick only has four priorities (High, Medium, Low, None). Please see Priority in [Task Format](#Task-format) for Task mapping of priority
+4. **Warning:** TickTickSync automatically creates .md files corresponding with the lists/projects created in TickTick and downloads all tasks to them.  
+   If the files are deleted in Obsidian while the TickTickSync plugin is active, those tasks will be deleted. If you accidentally confirm that deletion:
```
Comment (MandeepCodes) [inline README.md:21]: @thesamim Can you please make this path configurable. Where a user can specify a folder to store these .md files.

Comment (thesamim) [reply README.md:21]: On the roadmap: https://github.com/thesamim/TickTickSync/issues/12

**Review by thesamim — COMMENTED** [review]
Hope everybody reads the Readme first.

**Review by thesamim — COMMENTED** [review]
All Looks Good.

**Pull request merged**